### PR TITLE
samples: nrf9160: Make test builds run for nrf9160_pca10090ns

### PR DESCRIPTION
**Pull request by rlubos**
Samples should run in non-secure execution mode, hence should be built
for nrf9160_pca10090ns. Align overlays as well.

Signed-off-by: Robert Lubos <robert.lubos@nordicsemi.no>

### CODE REVIEW

**Comment by lemrey** [comment]
@rlubos it compiles, but I would like to get the documentation update in as well; I am working on it.

**Comment by lemrey** [comment]
@ru-fu I updated the documentation, please have a look.

File: samples/nrf9160/asset_tracker/README.rst [hunk 1]
```diff
@@ -68,7 +68,8 @@ Building and running
 
 This sample can be found under :file:`samples/nrf9160/asset_tracker` in the |NCS| folder structure.
 
-The sample is built as a non-secure firmware image, thus with ``CONFIG_TRUSTED_EXECUTION_NONSECURE=y`` set in ``prj.conf``.
+The sample is built as a non-secure firmware image, with ``CONFIG_TRUSTED_EXECUTION_NONSECURE=y`` set in ``prj.conf``.
```
[the remark this answers is not in the document]
Comment (lemrey) [reply samples/nrf9160/asset_tracker/README.rst]: Ok, we can simplify a bit.

**Comment by lemrey** [comment]
@carlescufi CI will need https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/418 to pass.

File: samples/nrf9160/asset_tracker/README.rst [hunk 2]
```diff
@@ -68,7 +68,7 @@ Building and running
 
 This sample can be found under :file:`samples/nrf9160/asset_tracker` in the |NCS| folder structure.
 
-The sample is built as a non-secure firmware image, thus with ``CONFIG_TRUSTED_EXECUTION_NONSECURE=y`` set in ``prj.conf``.
+The sample is built as a non-secure firmware image for the nrf9160_pca10090ns board.
```
Comment (ru-fu) [inline samples/nrf9160/asset_tracker/README.rst]: The same instruction is also in the readme file for the secure boot sample.

Comment (ru-fu) [reply samples/nrf9160/asset_tracker/README.rst]: And should that readme also contain info like "build the sample for nrf9160_pca10090 and the application with nrf9160_pca10090ns"?

Comment (lemrey) [reply samples/nrf9160/asset_tracker/README.rst]: I have addressed the first comment. Regarding the second, I don't think we need to document how to build the non-secure application in the `secure_boot` sample.

Comment (ru-fu) [reply samples/nrf9160/asset_tracker/README.rst]: But we do already - see line 49 in that readme (cannot comment there).

Comment (lemrey) [reply samples/nrf9160/asset_tracker/README.rst]: Should be fixed.

Comment (ioannisg) [reply samples/nrf9160/asset_tracker/README.rst]: now it is correct

File: samples/nrf9160/asset_tracker/prj.conf [hunk 3]
```diff
@@ -4,9 +4,6 @@ CONFIG_TEST_RANDOM_GENERATOR=y
 CONFIG_ASSERT=y
 CONFIG_REBOOT=y
 
-# Trusted execution
-CONFIG_TRUSTED_EXECUTION_NONSECURE=y
```
Comment (ioannisg) [inline samples/nrf9160/asset_tracker/prj.conf]: good

File: samples/nrf9160/secure_boot/README.rst [hunk 4]
```diff
@@ -46,7 +46,7 @@ Requirements for the application firmware
 * The application firmware must be located in the slot_ns flash partition.
   For more details, see the `nrf9160_pca10090_partition_conf.dts`_ file in the nrf9160_pca10090 board definition.
   Note that if you build your application firmware with the |NCS|, this requirement is automatically fulfilled.
-* The application firmware must be built as a non-secure firmware, which means that you must set ``CONFIG_TRUSTED_EXECUTION_NONSECURE=y`` in the ``prj.conf`` file of the application.
+* The application firmware must be built as a non-secure firmware for the nrf9160_pca10090ns board.
```
Comment (ioannisg) [inline samples/nrf9160/secure_boot/README.rst]: perhaps "for" --> "by selecting"?

**Pull request merged**